packaging/install.cmake

```diff
@@ -92,10 +92,13 @@ if(UNIX)
 
   # replace MPIEXEC MPIEXEC_NUMPROC_FLAG and MPI_IMPL KHIOPS_MPI_EXTRA_FLAG ADDITIONAL_ENV_VAR
   if("${MPI_IMPL}" STREQUAL "openmpi")
-    set(KHIOPS_MPI_EXTRA_FLAG "--allow-run-as-root --quiet")
+    set(KHIOPS_MPI_EXTRA_FLAG "--allow-run-as-root")
+    set(KHIOPS_MPI_QUIET "--quiet")
     set(ADDITIONAL_ENV_VAR "export OMPI_MCA_btl_vader_single_copy_mechanism=none # issue on docker")
+    set(ADDITIONAL_ENV_VAR_DISPLAY "\techo OMPI_MCA_btl_vader_single_copy_mechanism none")
```

Comment:
Remove `\t` as `@ADDITIONAL_ENV_VAR_DISPLAY` is already indented in `khiops_env.in`.

Reply:
I am agree but as a matter of fact the `\t` is mandatory. I just checked it in my laptop. I think the cmake search&replace remove all blank characters.

Reply:
> I am agree but as a matter of fact the `\t` is mandatory. I just checked it in my laptop. I think the cmake search&replace remove all blank characters.

Then OK for 4 spaces instead of TAB.